README.md

```diff
@@ -20,22 +20,68 @@ small set of sample data trained with intents like these:
 * `weather`
 * `capabilities`
 
-To see all the intents, open [`/training/car_intents.csv`](https://raw.githubusercontent.com/watson-developer-cloud/conversation-simple/master/training/car_intents.csv).
-
 These intents help the system to understand variations of questions and commands that you might submit.
 For example, if you say *"Wipers on"* or *"I want to turn on the windshield wipers"*, the system
 understands that in both cases your intent is the same and responds accordingly.
 
-## Getting Started
-If you used the **Deploy to Bluemix** button, skip this section and go to **Creating a workspace**.  
+<a name="bluemix">
+# Getting Started using Bluemix
+</a>
+
+## Before you begin
+1 Ensure that you have a [Bluemix account](https://console.ng.bluemix.net/registration/).
+
+2 Ensure that you have the necessary space available in your Bluemix account. This action deploys 1 application and 3 services.
+   * You can view this on your Bluemix Dashboard. Tiles will show what space you have available.
+   * For example, for Services & APIS
+
+    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;![](readme_images/services.PNG)
```

Comment:
@germanattanasio @sgregglives 
It's the only way I've found so far to move the button images where they are. If you have other suggestions, I'm MORE than happy to hear them. I've scoured forums looking for a way to do this formatting, but markdown wasn't really meant for styling -- in fact, if this type of styling is something we want to continue, then git/markdown is not the way to go. 

I know we could apply a CSS to each README. Of course that then requires using CSS...and another document to be edited when we make changes. 

Thoughts?